### ISSUE TITLE
dateTimeFormatString in multiple columns

### ISSUE THREAD

**Issue by earnaud** [EDIorg/EMLassemblyline#52]
Hi EAL team ! 

Here is a common issue: what would be best practices to describe dateTimeFormat stored in multiple columns of a dataset? e.g. 1st column "YEAR", 2nd column "MONTH" and 3rd column "DAY". 

This question is also valid for MetaShARK's work.

**Comment by clnsmth** [comment]
One option is to assign a format string for each column and convert them to "date" storageType in EML.

**Comment by earnaud** [comment]
Ok. By now, MetaShARK keeps attribute's class as it is guessed by default and does not allow the user to change it. I will change this soon (see [there](https://github.com/earnaud/MetaShARK-v2/issues/16) )

**Comment by clnsmth** [comment]
Yeah, there's room for improvement in EAL's class guessing functions (PRs are welcome). 

Allowing the user to manually edit attribute classes in MetaShARK would be great!
